qabelbox/src/main/java/de/qabel/qabelbox/helper/FileHelper.java

```diff
@@ -28,4 +32,28 @@ public static String readFileAsText(FileInputStream fis) throws IOException {
         }
         return fileContent.toString();
     }
+    public static String readInputStreamAsText(InputStream fis) throws IOException {
```

Comment:
These methods are not called, so I'm not sure where they should read their input from. If they should read files from the flash storage, a buffer of 4096 bytes should be considered because I know from personal experience that is is much faster (flash block size is usually 4096 bytes). This size is usually used in all IO libraries I know. (e.g. Apache [IOUtils](https://commons.apache.org/proper/commons-io/apidocs/src-html/org/apache/commons/io/IOUtils.html) line 134)